Updating to Teams Admin Center Instructions (#6138)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/change-the-phone-numbers-on-your-audio-conferencing-bridge.md

In addition to the phone numbers already assigned to your conferencing bridge, you can [get additional service numbers](/microsoftteams/getting-service-phone-numbers) (toll and toll-free numbers used for audio conferencing) from other locations, and then assign them to the conferencing bridge so you can expand coverage for your users.
  
> [!NOTE]

> > To be able to assign/unassign a phone number for a conferencing bridge, the phone number must be a '*service*' number. You can see the type of number it is by navigating to **Voice** > **Phone numbers** in the Microsoft Teams admin center and looking in the **Number Type** column. Microsoft 365 or Office 365 Communications Credits must be set up first in order for users to dial into the bridge on a toll free number.

Reviewer: ```suggestion
> To be able to assign/unassign a phone number for a conferencing bridge, the phone number must be a '*service*' number. You can see the type of number it is by navigating to **Voice** > **Phone numbers** in the Microsoft Teams admin center and looking in the **Number Type** column. Microsoft 365 or Office 365 Communications Credits must be set up first in order for users to dial into the bridge on a toll-free number.
```

## Steps when you are assigning a new service phone number to your conference bridge

### Step 1 - Assign the new phone number to your audio conferencing bridge

![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**

> 1. In the left navigation, go to **Voice** > **Phone numbers**.

Reviewer: ```suggestion
1. On the left navigation pane, go to **Voice** > **Phone numbers**.
```

2. Select the phone number from the list, and click **Edit**.

> 3. On the **Edit** page, under **Assigned to** click the dropdown and then click to select **Conference bridge**, and click **Apply**.

Reviewer: ```suggestion
3. On the **Edit** page, under **Assigned to**, click the dropdown and then select **Conference bridge** > **Apply**.
```

### Step 2 - Change the default phone number of your conference bridge (optional)

The default phone number of your conference bridge defines the caller ID that will be used when an outbound call is placed by a participant or the organizer from within a meeting.

Only a service toll number can be set as the default number for your conferencing bridge; **service toll-free numbers can't be set as the default number of your conferencing bridge**. If you are assigning a service toll number and you would like to set it as the new default number for your audio conferencing bridge, perform these steps:

![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**

> 1. In the left navigation, go to **Meetings** > **Conference bridges**.

Reviewer: ```suggestion
1. On the left navigation pane, go to **Meetings** > **Conference bridges**.
```

2. Highlight the service toll number that you want to configure as the default.

3. Select **Set as default**.
 
### Step 3 - Change the default phone numbers that are included in the meeting invites of users (optional)

The default phone numbers of a user are the ones that are included on their meeting invites when they schedule a meeting. For more information, including how the default phone numbers are assigned for new users, see [Set the phone numbers included on invites in Microsoft Teams](set-the-phone-numbers-included-on-invites-in-teams.md) or [Set the phone numbers included on invites in Skype for Business Online](/SkypeForBusiness/audio-conferencing-in-office-365/set-the-phone-numbers-included-on-invites).

![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**

> 1. In the left navigation, go to **Users** and click the Display name of the desired user on the list.

Reviewer: ```suggestion
1. On the left navigation pane, go to **Users** and click the Display name of the desired user on the list.
```

  
### Step 3 - Unassign the old phone number from the audio conferencing bridge

![An icon showing the Microsoft Teams logo](media/teams-logo-30x30.png) **Using the Microsoft Teams admin center**

1. In the left navigation, go to **Voice** > **Phone numbers**.

> 2. If the phone number is a toll-free number, select the phone number from the list, and click **Release**. If the phone number is a toll-number, please contact [Microsoft support](https://go.microsoft.com/fwlink/?linkid=2091806) to have the phone number unassigned.

Reviewer: ```suggestion
2. If the phone number is a toll-free number, select the phone number from the list, and click **Release**. If the phone number is a toll number, please contact [Microsoft support](https://go.microsoft.com/fwlink/?linkid=2091806) to have the phone number unassigned.
```
